tests/integration/test_github.py

```diff
@@ -414,6 +413,103 @@ def test_update_pr_info(self):
         assert pr_info.title == orig_title
         assert pr_info.description == orig_description
 
+    def _pr_close_temp(self, pr_id):
+        """
+        Temporary function, should be replaced by GithubProject.pr_close(), once implemented
+        """
+        pr = self.hello_world_project.github_repo.get_pull(int(pr_id))
+        pr.edit(state="closed")
```

Comment:
@lachmanfrantisek want to cleanup this temp method, because #273 is merged
is it possible to do in this pull-request - reopen it , or need i create new one?

Reply:
You need to create a new one. Thanks for that!

Reply:
np will do it